interfaces/Portal/src/app/pages/home/components/programs-list/programs-list.component.ts

```diff
@@ -27,10 +27,15 @@ export class ProgramsListComponent implements OnInit {
 
   private async refresh() {
     this.loading = true;
-    const programs = await this.programsService.getAllPrograms();
+    const programIds = await this.programsService.getAllProgramIds();
     this.programStats = await this.programsService.getAllProgramsStats(
-      programs.map((p) => p.id),
+      programIds.map((id) => id),
```

Comment:
Can't this simply be like this?

```suggestion
      programIds
```

Reply:
Hahaha yes 😄